styles/components/_availability-table.scss

```diff
@@ -30,6 +30,7 @@
     width: 42px;
     margin-bottom: 20px;
     margin-left: -20px;
+    margin-top: 50px;
```

Comment:
can you use `$very-large-space` for margin top?
I know 20px was there before. it'll be great if you can change those to use `$medium-space`